pex/targets.py

```diff
@@ -136,6 +136,9 @@ def __str__(self):
         # type: () -> str
         return str(self.platform.tag)
 
+    def render_description(self):
+        raise NotImplementedError()
```

Comment:
Only because enough situations rebuff usage that I just no longer use at all. Here its the attr.s class decorator that foils MyPy when combined with ABC / vice-versa. If you don't use ABC and you just `@abstractmethod`, MyPy is fooled.

To boot, this is a sealed class. Type checking is great, but if you need it for code local to 1 file, you're moving too fast.